test/unit/test_default_inference_handler.py

```diff
@@ -166,6 +166,13 @@ def test_default_output_fn_csv_float(inference_handler):
     assert '1.0,2.0,3.0\n4.0,5.0,6.0\n'.encode("utf-8") == output
 
 
+def test_default_output_fn_multiple_content_types(inference_handler, tensor):
+    accept = ", ".join(["application/unsupported", content_types.JSON, content_types.CSV])
+    output = inference_handler.default_output_fn(tensor, accept)
+
+    assert json.dumps(tensor.cpu().numpy().tolist()) == output
```

Comment:
The `output_fn` automatically converts Tensors to NumPy arrays, so the `tensor.cpu().numpy().tolist()` is unnecessary. I've kept it in for consistency with the other tests.

Reply:
Yeah, I think the `tensor.cpu().numpy().tolist()` is unnecessary given here: https://github.com/aws/sagemaker-pytorch-inference-toolkit/blob/master/src/sagemaker_pytorch_serving_container/default_pytorch_inference_handler.py#L103

Have you tried removing the conversion for other default handler tests? If it works, we should remove it from test cases.

Reply:
I realize I misread the code. The `tensor.cpu().numpy().tolist()` is passed to `json.dumps`, not `default_output_fn`. The test code is fine as-is.